### ISSUE TITLE
what's class_names?

### ISSUE THREAD

**Issue by davvvy** [ziqi-jin/finetune-anything#14]
Thanks for sharing this material first.

According to this line(https://github.com/ziqi-jin/finetune-anything/blob/main/extend_sam/runner.py#L97), 
class_names = self.val_loader.dataset.class_names.
But I can't find class_names in config file. 

Is it a list consisting of 19 semantic labels? Which labels does it have?
Could you please give an example of class_names?

**Comment by ziqi-jin** [comment]
Now the BaseSemanticDataset and TorchVOCSegmentation is available, if you want to use BaseSemanticDataset, you should add the 'class_names' into 'metainfo' as below,  and if your img and annotation folders' names are not 'img' and 'ann', you also shoult modify the 'prefix' as below
`
    metainfo = {'class_names': ['bicycle', 'bird', 'boat', 'bottle',
                            'bus', 'car', 'cat', 'chair', 'cow',
                            'diningtable', 'dog', 'horse', 'motorbike', 'person',
                            'pottedplant', 'sheep', 'sofa', 'train', 'tvmonitor']}
    prefix = {'img_path':'your_img_folder', 'ann_path':"your_ann_folder"}
    dataset = BaseSemanticDataset(metainfo=metainfo,dataset_dir='/data/jinziqi/DATASETS/VOCdevkit/VOC2012/',
                                  img_suffix='.png', ann_suffix='.jpg', data_prefix=prefix, transform=transform, target_transform=transform)
`
this example shows the VOC dataset classes, if you have DIY dataset, just add your own class names into metainfo

**Comment by ziqi-jin** [comment]
I close this issue, if you have other questions, open this issue.
